Mods/SML/Source/SML/Private/Registry/ModContentRegistry.cpp

```diff
@@ -906,8 +909,16 @@ bool UModContentRegistry::IsDescriptorFilteredOut( const UObject* ItemDescriptor
 	if (!EnumHasAnyFlags(Flags, EGetObtainableItemDescriptorsFlags::IncludeVehicles) && descriptorClass->IsChildOf<UFGVehicleDescriptor>()) {
 		return true;
 	}
-	if (!EnumHasAnyFlags(Flags, EGetObtainableItemDescriptorsFlags::IncludeSpecial) && ( descriptorClass->IsChildOf<UFGWildCardDescriptor>() || descriptorClass->IsChildOf<UFGAnyUndefinedDescriptor>() || descriptorClass->IsChildOf<UFGOverflowDescriptor>() || descriptorClass->IsChildOf<UFGNoneDescriptor>() )) {
-		return true;
+	if (!EnumHasAnyFlags(Flags, EGetObtainableItemDescriptorsFlags::IncludeSpecial)) {
+		if (descriptorClass->IsChildOf<UFGWildCardDescriptor>() || descriptorClass->IsChildOf<UFGAnyUndefinedDescriptor>() || descriptorClass->IsChildOf<UFGOverflowDescriptor>() || descriptorClass->IsChildOf<UFGNoneDescriptor>()) {
+			return true;
+		}
+		if (descriptorClass->ImplementsInterface(USMLExtendedAttributeProvider::StaticClass())) {
+			UObject* ItemDescriptorCDO = descriptorClass->GetDefaultObject();
+			const auto ItemTags = ISMLExtendedAttributeProvider::Execute_GetGameplayTagsContainer(ItemDescriptorCDO);
+			const auto SmlSpecialTag = FGameplayTag::RequestGameplayTag("SML.Registry.Item.SpecialItemDescriptor", true);
```

Comment:
Since SML has c++, and this tag is used in c++, it makes more sense to register it using native tags, with the new UE5 macros, rather than registering it in a config file and using the find function every time.

In the .h: `SML_API UE_DECLARE_GAMEPLAY_TAG_EXTERN(TAG_SML_Registry_Item_SpecialItemDescriptor)`
In the .cpp: `UE_DEFINE_GAMEPLAY_TAG_COMMENT(TAG_SML_Registry_Item_SpecialItemDescriptor, "SML.Registry.Item.SpecialItemDescriptor", "comment goes here")`
`return ItemTags.HasTag(TAG_SML_Registry_Item_SpecialItemDescriptor);`

Reply:
Sounds good, but we should then add a tag to examplemod using the config file approach. Any ideas for what that tag could be?